### ISSUE TITLE
Accessibility: talkback speaks random "close" button

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#618]
**Description**  
The very first element on the main feed page reads as "close" which when tapped expands the FAB. 

**How to Reproduce**  
1. Go to system settings > accessibility > talkback and turn it on
2. Launch Thunder (fresh, not from background)
3. See error

OR
1. Go to system settings > accessibility > talkback and turn it ON
2. Launch Thunder
3. Tap hamburger menu at top left
4. Flick from right to left in order to go back one element
5. See error

**Expected Behavior**  
Talkback should not see this since the FAB is not open. The first element focused should be the hamburger menu

**Screenshots**  

**Device & App Version:**  
 - Device: [e.g. iPhone6] OnePlus Nord N200
 - OS: [e.g. iOS8.1] Android 13
 - Version [e.g. 22] v0.2.1-alpha.3+14

**Additional Context**
